m4/pbs_default_scheduler.m4

```diff
@@ -37,12 +37,7 @@
 
 AC_DEFUN([PBS_AC_DEFAULT_SCHEDULER],
 [
-  AC_MSG_CHECKING([for Default scheduler name])
```

Comment:
I agree with @bhroam here. If we don't care to change the value at configure time, then the macro should be made static with no substitution performed. We should not set things up so that pbs_share.h is updated by the autotools, that's what /src/include/pbs_config.h is for, IMHO.